pxc/jenkins/qa_pxc_cross-pipeline.yml

```diff
@@ -0,0 +1,47 @@
+- job:
+    name: qa_pxc_cross-pipeline
```

Comment:
The job name must be meaningful, something like 
eg
qa_pxc_cross_version_57_80_test

Reply:
Fixed !